assets/js/modules/analytics/components/dashboard/DashboardUniqueVisitorsWidget.js

```diff
@@ -55,35 +55,38 @@ function DashboardUniqueVisitorsWidget() {
 		const accountID = store.getAccountID();
 		const profileID = store.getProfileID();
 		const internalWebPropertyID = store.getInternalWebPropertyID();
-		const commonArgs = {
-			dateRange: select( CORE_USER ).getDateRange(),
-		};
 
 		const url = select( CORE_SITE ).getCurrentEntityURL();
-		if ( url ) {
-			commonArgs.url = url;
-		}
+		const { compareStartDate, compareEndDate, startDate, endDate } = select( CORE_USER ).getDateRangeDates( { offsetDays: DATE_RANGE_OFFSET, compare: true } );
+
 		const sparklineArgs = {
+			compareStartDate,
+			compareEndDate,
+			startDate,
+			endDate,
+			url,
```

Comment:
Adding this in here unconditionally is an unrelated change against how it was before. The args should only include `url` if it is a truthy value.